be/src/vec/columns/column_string.cpp

```diff
@@ -134,34 +133,41 @@ void ColumnStr<T>::insert_range_from(const IColumn& src, size_t start, size_t le
     if (length == 0) {
         return;
     }
+    auto do_insert = [&](const auto& src_concrete) {
```

Comment:
clone_resized  method should check if to size is larger than limit

Reply:
if `to_size` is bigger than `from_size`, extra element is empty string, total stringg size will not bigger than src size.